packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/siw.md

```diff
@@ -1,22 +1,20 @@
 ### 1: Initiate password recovery
 
-To begin the password recovery flow, the user must
+The user starts the password recovery flow with the following steps:
 
-1. Click the **Forgot Password?** link on the sign-in page.
-2. Enter their **Email or Username** in the box and click **Next**.
-3. Choose **Email** as the authenticator they want to use for password recovery and click **Submit**.
+1. Clicks the **Forgot Password?** link on the sign-in page.
+2. Enters their **Email or Username** in the dialog and then clicks **Next**.
+3. Chooses **Email** as the authenticator they want to use for password recovery and clicks **Submit**.
 
 Okta then tells the user to either click the link in the email or enter the code to continue and sends an email to their email address matching the Forgot Password template that was altered earlier.
 
 <div class="common-image-format">
 
-![Screenshot of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png "Password Recovery Email")
+![Example of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png "Password recovery email")
 
 </div>
 
-The **Reset Password** link in the email includes the `OTP` and `request.relayState` variables as query parameters back to the application. For example,
-
-`https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}` becomes `https://localhost:44314/magiclink/callback?otp=726009&state=1b34371af02dd31d2bc4c48a3607cd32`.
+The email's **Reset Password** link includes the `otp` and `request.relayState` variables sent back as query parameters to the application. For instance, the URL in the email template,  `http://localhost:8080/magic-link/callback?otp=${oneTimePassword}&state=${request.relayState}`, might be rendered as `http://localhost:8080/magic-link/callback?otp=726009&state=1b34371af02dd31d2bc4c48a3607cd32` in the email sent to the user.
```

Comment:
This is a repeat of the text in a document above... is this supposed to be here? If it is, same recommended revision:
This is a bit unclear. I also don't think the reference to Reset Passowrd link needs to be the proper noun... it a generic thing in this case, not a specific button in an email. How about:
```suggestion
The reset password link in the email link includes variables for the `otp` and `request.relayState` which are sent in query parameters to ?your/the? application. For example, in the email setnt to the user, the URL `http://localhost:8080/magic-link/callback?otp=726009&state=1b34371af02dd31d2bc4c48a3607cd32` results from an email template URL of `http://localhost:8080/magic-link/callback?otp=${oneTimePassword}&state=${request.relayState}`, an OTP of `726009`, and a relay state of `1b34371af02dd31d2bc4c48a3607cd32`.
```